From 406d8e37bfd731a35e4440871a6794db05d1730c Mon Sep 17 00:00:00 2001
From: Martin Basti <mbasti@redhat.com>
Date: Mon, 21 Dec 2015 13:15:28 +0100
Subject: [PATCH] WebUI: fix ipa_error.css

<h2> had improper CSS style that causes that <h2> was placed on  wrong position
---
 install/html/ipa_error.css | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/install/html/ipa_error.css b/install/html/ipa_error.css
index 6cf70d885d80c1a93627d1acb231c62018b1bd15..d64eef278b4af32008f35b8cd696f18249522ce8 100644
--- a/install/html/ipa_error.css
+++ b/install/html/ipa_error.css
@@ -9,16 +9,6 @@
 
 /* ssbrowser */
 
-.ssbrowser h2 {
-    font-family: 'Overpass Bold';
-    font-size: 1em;
-    font-weight: bold;
-    margin-bottom: 3em;
-    margin-left: 5em;
-    margin-top: -3em;
-}
-
-
 .ssbrowser h3 {
     font-size: 1.1em;
     margin-bottom: 1em;
-- 
2.6.4

